src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -352,6 +352,21 @@ def download_must_gather(kubeconfig: str, dest_dir: str):
 
     log.debug("Removing must-gather directory %s after we archived it", must_gather_dir)
     shutil.rmtree(must_gather_dir)
+    describe_cluster_operators(kubeconfig, dest_dir)
```

Comment:
I think it shouldn't be called from ``download_must_gather`` as it's doing something else (especially that it has ``JunitTestCase`` decorator which should be inside of a ``JunitTestCase`` afaik)

Reply:
Hmm, I think the name is still describing the main thing this function does (getting the co status is informative but just has very little info compared to must-gather logs).
I added an optional parameter (describe_cluster_operators) to the function that indicates that it will optionally run that command as well).
I removed the `JunitTestCase` decorator from the `describe_cluster_operators`